components/scream/cime_config/buildnml

```diff
@@ -47,7 +47,7 @@ import yaml
 logger = logging.getLogger(__name__)
 
 CIME_VAR_RE = re.compile(r'[$][{](\w+)[}]')
-SWITCH_RE   = re.compile(r'<\s*(\w+)\s*:')
+SWITCH_RE   = re.compile(r'<\s*([^:]+)\s*:')
```

Comment:
This is the only change in the whole PR that I could see even possibly touching SCREAMv0 tests.  @jgfouca and/or @brhillman .  Do you have any thoughts on what is causing the DIFFs when we run the AT?

Reply:
@AaronDonahue , scream is not currently using this syntax in its YAML files. This change should not be causing DIFFs.

Reply:
I see a new test was added. That would definitely cause a DIFF (no baselines).